src/main/java/randoop/reflection/OperationModel.java

```diff
@@ -579,7 +589,13 @@ private void addClassTypes(
     mgr.add(new TypeExtractor(this.inputTypes, accessibility));
     mgr.add(new TestValueExtractor(this.annotatedTestValues));
     mgr.add(new CheckRepExtractor(this.contracts));
-    if (literalsFileList.contains("CLASSES")) {
+
+    // Extract literals from classes under test. Two modes are mutually exclusive:
+    // 1. constant_tfidf: uses TF-IDF to select constants (stores in scopeToConstantStatistics)
+    // 2. literals_file="CLASSES": simple extraction without statistics (stores in classLiteralMap)
```

Comment:
The constants discovered or extracted should be identical.  constant-tfidf should only change which constants are chosen, not which constants have a chance of being chosen.

A simple way to do that is to ensure that, when determining the constants, they should be put in a single data structure (scopeToConstantStatistics is fine) that is used thereafter.  Not all uses of constants need all the information in scopeToConstantStatistics, but that extra information is small and does no harm.  For example, normal selection (not constant-tfidf) would just copy information from the data structure to the pool.